app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,56 @@
+class AlbumsController < ApplicationController
+  def index
+    @media = Album.order(ranking: :desc)
+  end
+
+  def show
+    @media = Album.find(params[:id])
```

Comment:
We can DRY this out a bit by making this line into a `before_action`, because it is used to setup the `@media` variable the same way in multiple controller actions.